integration-tests/netty4-http/src/main/java/org/apache/camel/quarkus/component/netty4/http/CamelRoute.java

```diff
@@ -19,10 +19,16 @@
 import org.apache.camel.builder.RouteBuilder;
 
 public class CamelRoute extends RouteBuilder {
+
+    public static final int PRODUCER_TARGET_PORT = 10102;
```

Comment:
Probably we'll need to re-use the AvailablePortFinder from Camel

Reply:
Not sure how? The test app AND the test classes need to agree on the same port number while they live in distinct JVM processes. We'd have to pass the random port via an endpoint running on some constant port - which I think is too complicated for what it brings. Or you maybe have a better idea?